test-configs.yaml

```diff
@@ -1024,6 +1024,12 @@ device_types:
     boot_method: uboot
     flags: ['big_endian']
 
+  sun50i-h5-nanopi-neo-plus2:
```

Comment:
Please follow alphabetical order - `nanopi` comes after `libretech`.

Reply:
sorry, that's weird I have inverted those two by saying "let put them in alphabetical order".